projects/vdk-core/src/vdk/api/plugin/plugin_input.py

```diff
@@ -198,6 +202,40 @@ def ingest_payload(
             invocations belong to same collection. Defaults to "data_job_name|OpID",
             meaning all method invocations from a data job run will belong to the
             same collection.
+
+        :return: [Optional] IngestionResult, containing plugin-defined
+        information about the result from the ingestion operation. This result
+        could be used for post-ingestion processing operations like sending
+        telemetry data about the ingested payload.
+
+        :exceptions: Exceptions raised while ingesting data can either be
+        handled at plugin level, or they will be automatically caught at the
+        vdk-core level.
+        """
+        pass
+
+    def post_ingest_process(
+        self,
+        payload: Optional[List[dict]] = None,
+        ingestion_result: Optional[IngestionResult] = None,
+        exceptions_list: Optional[List] = None,
+    ) -> None:
+        """
+        Do post-ingestion processing of the ingestion payload
```

Comment:
Provide an example in the pydoc of how it can be used - simple implementation of the method.

Reply:
Done, thanks